acceptance/bundle/variables/var_file_overrides/databricks.yml

```diff
@@ -0,0 +1,35 @@
+bundle:
+  name: TestResolveVariablesFromFile
+
+variables:
+  cluster:
+    type: "complex"
+  cluster_key:
+  cluster_workers:
+
+resources:
+  jobs:
+    job1:
+      job_clusters:
+        - job_cluster_key: ${var.cluster_key}
+          new_cluster:
+            node_type_id: "${var.cluster.node_type_id}"
+            num_workers: ${var.cluster_workers}
+
+targets:
+  with-defaults:
+    default: true
+    variables:
+      cluster_workers:
+        default: 1
+      cluster:
+        type: "complex"
+        default:
+          node_type_id: "default"
+      cluster_key:
+        default: "default"
+
+  without-defaults:
+
+  # see .databricks/bundle/default_target/ for variable values
+  with-default-variable-file:
```

Comment:
Why do you need a target here? The default variables are read for any target, right? so it's the same as without-defaults.